src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/DotnetCliHelper.cs

```diff
@@ -17,7 +15,6 @@ namespace Microsoft.CodeAnalysis.LanguageServer;
 internal sealed class DotnetCliHelper
 {
     internal const string DotnetRootEnvVar = "DOTNET_ROOT";
-    internal const string DotnetRootUserEnvVar = "DOTNET_ROOT_USER";
```

Comment:
🤔So the Client can't no longer config the runtime using this?

Reply:
Yeah -  that was the recommendation in the linked runtime issue.

Reply:
https://github.com/dotnet/vscode-csharp/blob/2d1e6cbc96b32ad8936903f40162cfd31bf576a8/package.json#L1328
So this is needed to be removed after this change, is that right?

Reply:
No - that settings allows you to override the *runtime* that the server itself starts with, which is different from the dotnet CLI we use to run tests. 

Previously O# allowed you to specify a custom dotnet CLI to run tests on using the cliPaths option
https://github.com/dotnet/vscode-csharp/blob/2d1e6cbc96b32ad8936903f40162cfd31bf576a8/package.json#L1294
We could potentially bring that back if people want it.

Reply:
Ah I mis-understand things here